opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -1835,6 +1837,13 @@ class InvestigationGraphComponent extends Component {
       this.fetchObjectRelCounts(newElements);
     }
     if (newElementsIds.length > 0) {
+      setStackDataInSessionStorage({ valueToStore: {
+        dateTime: new Date().getTime(),
+        investigatedEntitiesIdsList: newElementsIds,
+      },
+      key: 'preExpansionStateList',
+      stackValue: 10 });
+
```

Comment:
```suggestion
      setStackDataInSessionStorage({ 
        valueToStore: {
          dateTime: new Date().getTime(),
          investigatedEntitiesIdsList: newElementsIds,
        },
        key: 'preExpansionStateList',
        stackValue: 10,
      });

```

Did you configure your linter "on save" ?